app/models/EmailAddress.scala

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models
+
+import java.time.ZonedDateTime
+
+import play.api.libs.json.Json
+
+case class EmailAddress(address: String, timestamp: ZonedDateTime)
```

Comment:
Maybe rename to 'VerifiedEmailAddress' as its more than just an address (and will be consistent to the SFUS frontend I'm writing :slightly_smiling_face: )

Reply:
:+1: